FlightReservation/android/app/src/main/java/com/flightreservation/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.horcrux.svg.SvgPackage;
```

Comment:
when adding a new dependency, it's a good practice leaving these kind of changes in an isolated commit.
Not mandatory, but makes easier the review 😄